src/MongoDB/Session.c

```diff
@@ -618,6 +627,30 @@ static HashTable* php_phongo_session_get_debug_info(zval* object, int* is_temp T
 		ADD_ASSOC_NULL_EX(&retval, "operationTime");
 	}
 
+	if (intern->client_session) {
+		uint32_t server_id = mongoc_client_session_get_server_id(intern->client_session);
+
+		if (server_id > 0) {
```

Comment:
Since `server_id` is unsigned, `if (server_id)` would suffice.

Reply:
Ah, working with strongly typed languages is always a pleasure. You just have to remember that they work a lot cleaner than PHP does often.